app/_layouts/plugins/configuration_examples.html

```diff
@@ -5,6 +5,13 @@
 
 <h2 id="configuration">Basic configuration examples</h2>
 
+The following configuration examples are for illustrative purposes only, and shouldn't be used as-is in production.
```

Comment:
Shouldn't we be updating the examples to be production ready?

Reply:
We should, but these examples won't be production ready for some time. We just don't have all the info. Also, I don't think that this basic examples page will ever serve that purpose, since it's currently limited to only one example with little context.

Reply:
I'm worried that this will cause concern for others. Can we show this message inside `{% if page.examples_description %}`? That allows us to only show it on specific plugins?

Reply:
Done.